### ISSUE TITLE
Include categories & tags (when added via functions) to PAGE SETTINGS

### ISSUE THREAD

**Issue by keramch** [elementor/elementor#12683]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I get frustrated with having to update in elementor, then go back to the dashboard and add other content like categories or tags, and save again. On a site that's on a slow server, this is particularly frustrating and unnecessarily time consuming. (yes of course I try to avoid slow sites!) 

**Describe the solution you'd like**
It would be extremely handy and helpful if the categories & tags metaboxes (in fact, I guess, ANY additional metaboxes) for PAGES could be handled in the elementor editor "page settings" rather than having to exit elementor and manage in the dashboard, thus requiring an additional save. I recognize that categories & tags on pages are not a WP default, so it could be tricky or illogical, but it sure would be great! 

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by d19dotca** [comment]
Yes, this would be fantastic. I was just about to file a feature request for this very thing. It'd be great to have the ability to use Elementor Finder to create a new post and straight from the editor of the post be able to set the category and tags for the new post. This would save the step of having to go back to the WordPress post editor to assign the categories and tags for the post. It's surprising to me that this isn't already a feature, honestly, given the focus the past year or so to keep things in the editor (i.e. Site Settings) without having to leave the Elementor editor. Hoping this will be added soon. 😄
